apps/backend/src/lib/controller/notification/telegram/client.ts

```diff
@@ -0,0 +1,114 @@
+import { Bot } from "grammy";
+import { iNotificationClient } from "../interfaces";
+import { PrismaClient } from "@prisma/client";
+
+export class TelegramNotificationClient implements iNotificationClient {
+  private bot: Bot | undefined;
+  private readonly maxRetries = 3;
+  private readonly retryDelay = 5000;
+  prismaClient: PrismaClient | undefined;
```

Comment:
Should it ever be undefined? If not, remove?

Reply:
By initializing prismaClient before the env var check you remove `undefined`. Will push.

Reply:
Addressed: https://github.com/cursive-team/connections/pull/154/commits/7ce98da4ba460144b48f1f3580d3490aed651771